src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,57 +1,135 @@
 import React from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+
+// eslint-disable-next-line max-len
+const defaultPicture = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+type Props = {
+  onAdd: (newMovie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({
+  onAdd,
+}) => {
+  const [newMovie, setNewMovie] = React.useState<Movie | null>(null);
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [error, setError] = React.useState(false);
+  const [query, setQuery] = React.useState('');
+
+  const handleAddMovie = () => {
+    if (newMovie) {
+      onAdd(newMovie);
+      setNewMovie(null);
+      setQuery('');
+    }
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value);
+    setError(false);
+  };
+
+  const handleFindMovie = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setIsLoading(true);
+    getMovie(query)
+      .then((movie) => {
+        if (movie.Response === 'True') {
+          setError(false);
+          const pictureUrl = movie.Poster === 'N/A'
+            ? defaultPicture
+            : movie.Poster;
+
+          setNewMovie({
+            title: movie.Title,
+            description: movie.Plot,
+            imgUrl: pictureUrl,
+            imdbUrl: `https://www.imdb.com/title/${movie.imdbID}`,
+            imdbId: movie.imdbID,
+          });
+        }
+
+        if (movie.Response === 'False') {
```

Comment:
Try to rewrite this way 👇 
```suggestion
        if (movie.Response) {
```